openshift-knative-operator/hack/010-hack-storage-version-migrator-name.patch

```diff
@@ -0,0 +1,66 @@
+diff --git a/openshift-knative-operator/cmd/operator/kodata/knative-eventing/1.1.0/6-eventing-post-install.yaml b/openshift-knative-operator/cmd/operator/kodata/knative-eventing/1.1.0/6-eventing-post-install.yaml
+index 947867cfe..5d86799c0 100644
+--- a/openshift-knative-operator/cmd/operator/kodata/knative-eventing/1.1.0/6-eventing-post-install.yaml
++++ b/openshift-knative-operator/cmd/operator/kodata/knative-eventing/1.1.0/6-eventing-post-install.yaml
+@@ -172,10 +172,10 @@ roleRef:
+ apiVersion: batch/v1
+ kind: Job
+ metadata:
+-  generateName: storage-version-migration-eventing-
++  generateName: strg-version-migration-eventing-
+   namespace: knative-eventing
+   labels:
+-    app: "storage-version-migration-eventing"
++    app: "strg-version-migration-eventing"
+     eventing.knative.dev/release: "v1.1.0"
+ spec:
+   ttlSecondsAfterFinished: 600
+@@ -183,7 +183,7 @@ spec:
+   template:
+     metadata:
+       labels:
+-        app: "storage-version-migration-eventing"
++        app: "strg-version-migration-eventing"
+         eventing.knative.dev/release: "v1.1.0"
+       annotations:
+         sidecar.istio.io/inject: "false"
+diff --git a/openshift-knative-operator/cmd/operator/kodata/knative-serving/1.1.2/4-serving-post-install-jobs.yaml b/openshift-knative-operator/cmd/operator/kodata/knative-serving/1.1.2/4-serving-post-install-jobs.yaml
+index c20f646c6..7698d235c 100644
+--- a/openshift-knative-operator/cmd/operator/kodata/knative-serving/1.1.2/4-serving-post-install-jobs.yaml
++++ b/openshift-knative-operator/cmd/operator/kodata/knative-serving/1.1.2/4-serving-post-install-jobs.yaml
+@@ -1,6 +1,6 @@
+ 
+ ---
+-# /tmp/tmp.Edjrsh4tnc/serving-storage-version-migration.yaml
++# /tmp/tmp.Edjrsh4tnc/serving-strg-version-migration.yaml
+ # Copyright 2020 The Knative Authors
+ #
+ # Licensed under the Apache License, Version 2.0 (the "License");
+@@ -18,12 +18,12 @@
+ apiVersion: batch/v1
+ kind: Job
+ metadata:
+-  generateName: storage-version-migration-serving-
++  generateName: strg-version-migration-serving-
```

Comment:
Could we simply change only the job names usign some suffix specific to the release here?

Reply:
I suppose we'd need a patch "template" like we do with CSV. But it should be possible.
We'd first create the patch from the template (by adding new stuff in the `release-files` make target), then we'd have the correct patch to be applied in `generated-files` make target.